apps/server/src/stores/runtimeState.ts

```diff
@@ -536,22 +533,35 @@ export function update(): UpdateResult {
         throw new Error('runtimeState.updateIfWaitingToRoll: invalid state received');
       }
     }
-
+    //account for offset
+    const offsetClock = runtimeState.clock + runtimeState.runtime.offset;
     runtimeState.timer.phase = TimerPhase.Pending;
-    runtimeState.timer.secondaryTimer = runtimeState._timer.secondaryTarget - runtimeState.clock;
-    return { hasTimerFinished: false, hasSecondaryTimerFinished: runtimeState.timer.secondaryTimer < 0 };
+    runtimeState.timer.secondaryTimer = runtimeState._timer.secondaryTarget - offsetClock;
+    return { hasTimerFinished: false, hasSecondaryTimerFinished: runtimeState.timer.secondaryTimer <= 0 };
   }
 }
 
-export function roll(rundown: OntimeRundown): { eventId: MaybeString; didStart: boolean } {
+export function roll(rundown: OntimeRundown, offset = 0): { eventId: MaybeString; didStart: boolean } {
+  if (runtimeState.timer.phase === TimerPhase.Overtime) {
```

Comment:
@cpvalente do you have any ideas here?
this stops the roll action form executing and doesn't clear out or stop the timer in any way